hc-nested-list-behavior.html

```diff
@@ -62,6 +62,21 @@
       'item-open-changed': 'itemOpenChanged'
     },
     /**
+     * will close each expanded tree node.
+     * Relies upon logic being in place to prevent two sibling nodes from being open concurrently.
+     *
+     * @param {Object} [parts = this.parts] - object containing all properties of each node
+     */
+    collapseTree(parts = this.parts) {
+      if (!parts) return;
+      const children = Object.keys(parts);
+      children.forEach(child => {
```

Comment:
Instead of `child` use `key` or `id`, `child` implies it's an object containing the data, not the key itself, I would recommend the same for `children` => `keys`